webapp/src/components/calendar/fullCalendar.tsx

```diff
@@ -96,11 +98,21 @@ const CalendarFullView = (props: Props): JSX.Element|null => {
     const myEventsList = useMemo(() => (
         cards.flatMap((card): EventInput[] => {
             const property = propsRegistry.get(dateDisplayProperty?.type || 'unknown')
+
             let dateFrom = new Date(card.createAt || 0)
             let dateTo = new Date(card.createAt || 0)
-            if (property.isDate && property.getDateFrom && property.getDateTo) {
-                dateFrom = property.getDateFrom(card.fields.properties[dateDisplayProperty?.id || ''], card)
-                dateTo = property.getDateTo(card.fields.properties[dateDisplayProperty?.id || ''], card)
+            if (property instanceof DatePropertyType) {
+                const dateFromValue = property.getDateFrom(card.fields.properties[dateDisplayProperty?.id || ''], card)
+                if (!dateFromValue) {
```

Comment:
Just to confirm that, is there the possibility of having a date that has a from and not a to value? I mean. Something like "not-range-date". In that case you have both anyway? If that is the case, it is right, if not, we can end up filtering here cases where you don't have ranges.

Reply:
It was correct because `property.getDateTo()` always returned value. If not a range it created one. `getDateTo()` also incremented the date by 1 day. this is because FullCalendar's "end" date is exclusive. So if you set end date on 22nd. It will cover the 21st, but not the 22nd. 

Because this is a full calendar issue, I moved the date increment to full calendar and `getDateTo()` returns the actual `To` date or undefined if there is no range.

Reply:
And...this is currently the only caller of `getDateFrom` and `getDateTo`.